plan/logical_plans.go

```diff
@@ -106,8 +106,9 @@ type LogicalJoin struct {
 	leftProperties  [][]*expression.Column
 	rightProperties [][]*expression.Column
 
-	// DefaultValues is only used for outer join, which stands for the default values when the outer table cannot find join partner
-	// instead of null padding.
+	// DefaultValues is only used for outer join, which is values the inner table's  should be when the outer table
+	// doesn't match any inner table'row.
+	// That it's nil just means the default values is a slice of NULL.
```

Comment:
how about: 
```go
// DefaultValues are only used for "LeftOuterJoin" and "RightOuterJoin" as the inner row when there is no inner row in the inner table can be joined with an outer row.
// NOTE: It is only set during the optimization of "aggregate push down", otherwise it will be nil, witch means a slice of NULLs.
```